### PR TITLE
Added some dummy root page content, basic adaptive layout with navbar and favicon

### PR DESCRIPTION
**Pull request by BeOff**
# [Link to ticket](https://trello.com/c/sMIfJaEi/32-root-page-with-login-register-information)
## What it does:
- creates root entry in Rails routes
- adds some dummy root page content
- sets basic adaptive layout with navbar up
- provides favicon image and tag
- bans CoffeeScript
- tunes generators in application.rb
- removes some unwanted files like old backups and empty specs
- adds basic request specs for root page

## UI changes:
see above

### CODE REVIEW

**Comment by labs-ruby** [comment]
1. My suggestion is to add so called `StaticPagesController`, with one action `#root` for now.
In this particular situation in will work well for us, in future we might want to add more static pages like `FAQ` or `Contact Us`
2. Remove helpers if they are not needed
3. Remove empty coffee and scss files if they are not needed
4. Add basic request specs for root page to make sure that it returns 200OK

**Comment by BeOff** [comment]
[Have done requested changes](https://github.com/labs-ruby/places-project/pull/3#issuecomment-496117388)

File: spec/requests/root_page_spec.rb [hunk 1]
```diff
@@ -0,0 +1,9 @@
+require 'rails_helper'
+
+RSpec.describe "Root page", type: :request do
+  before { get '/' }
+  it { expect(response).to have_http_status 200 }
+  it { expect(response.body).to include 'Sign In' }
+  it { expect(response).to render_template(:root) }
+  it { expect(response).not_to redirect_to('http://www.example.com') }
```
Comment (labs-ruby) [inline spec/requests/root_page_spec.rb]: Useless expectation

File: app/helpers/root_page_helper.rb [hunk 2]
```diff
@@ -0,0 +1,2 @@
+module RootPageHelper
```
Comment (labs-ruby) [inline app/helpers/root_page_helper.rb]: Remove this helper, it's empty

Comment (BeOff) [reply app/helpers/root_page_helper.rb]: Removed it.

File: spec/requests/root_page_spec.rb (hunk 1 above)
Comment (BeOff) [reply spec/requests/root_page_spec.rb]: Removed it.

**Pull request merged**